package.json

```diff
@@ -110,7 +110,9 @@
       "eslint:recommended",
```

Comment:
Since this PR is about linting I would enforce it using "plugin:vue/recommended" instead of plugin:vue/essential as in  
https://github.com/ICIJ/datashare-client/blob/master/.eslintrc.js